### PR TITLE
Add NCTL docker test case

### PR DESCRIPTION
**Pull request by gyroflaw**

### CODE REVIEW

File: test/nctl/setup.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    '11101:11101',
+    '-p',
+    '14101:14101',
+    '-p',
+    '18101:18101',
+    '--env',
+    'PREDEFINED_ACCOUNTS=true',
+    '--env',
+    'DEPLOY_DELAY=0sec',
+    'makesoftware/casper-nctl'
```
Comment (TomVasile) [inline test/nctl/setup.ts:23]: I think relying on a third party image will create problems for us

File: test/nctl/NCTL.test.ts [hunk 2]
```diff
@@ -148,10 +196,10 @@ describe('RPC', () => {
   });
 
   // TODO: Deploys required
-  it('chain_get_block_transfers - blockHash', async () => {
-    const transfers = await client.getBlockTransfers(exampleBlockHash);
-    expect(transfers).to.be.an.instanceof(Transfers);
-  });
+  // it('chain_get_block_transfers - blockHash', async () => {
```
Comment (TomVasile) [inline test/nctl/NCTL.test.ts:199]: should this test be commented out?

**Review by TomVasile — COMMENTED** [review]
I have a concern that using a third party image here will cause us issue down the road. 

What was the need for running this test in an nctl network? The current test seems to work fine on the single node process.

**Comment by gyroflaw** [comment]
Closing this PR due to [Docker anti-patterns](https://codefresh.io/blog/docker-anti-patterns/)

**Pull request closed without merging**